File: projetoES3/src/model/domain/Telefone.java
package model.domain;

public class Telefone extends EntidadeDominio{

	private String tipoTelefone;
	private String dddTelefone;
	private String numeroTelefone;
	
	public Telefone(String tipoTelefone, String dddTelefone, String numeroTelefone) {
		this.tipoTelefone = tipoTelefone;
		this.dddTelefone = dddTelefone;
		this.numeroTelefone = numeroTelefone;
	}

	public String getTipoTelefone() {
		return tipoTelefone;
	}

	public void setTipoTelefone(String tipoTelefone) {
		this.tipoTelefone = tipoTelefone;
	}

	public String getDddTelefone() {
		return dddTelefone;
	}

	public void setDddTelefone(String dddTelefone) {
		this.dddTelefone = dddTelefone;
	}

	public String getNumeroTelefone() {
		return numeroTelefone;
	}

	public void setNumeroTelefone(String numeroTelefone) {
		this.numeroTelefone = numeroTelefone;
	}

	
	

}

File: typeScript/src/model/domain/Telefone.ts
import {EntidadeDominio} from "./EntidadeDominio";
import {Fornecedor} from "./Fornecedor";

export class Telefone extends EntidadeDominio {
    constructor(
        private tipoTelefone: string,
        private dddTelefone: string,
        private numeroTelefone: string,
        private fornecedorId: string
    ) {
        super();
    }

    public setFornecedorId(id: string) {
        this.fornecedorId = id
    }

    public getFornecedorId() {
        return this.fornecedorId
    }

    getTipoTelefone(): string {
        return this.tipoTelefone;
    }

    setTipoTelefone(value: string) {
        this.tipoTelefone = value;
    }

    getDddTelefone(): string {
        return this.dddTelefone;
    }

    setDddTelefone(value: string) {
        this.dddTelefone = value;
    }

    getNumeroTelefone(): string {
        return this.numeroTelefone;
    }

    setNumeroTelefone(value: string) {
        this.numeroTelefone = value;
    }


}